apps/dashboard/src/components/workflow-list-empty.tsx

```diff
@@ -3,13 +3,22 @@ import { VersionControlDev } from '@/components/icons/version-control-dev';
 import { VersionControlProd } from '@/components/icons/version-control-prod';
 import { Button } from '@/components/primitives/button';
 import { useEnvironment } from '@/context/environment/hooks';
-import { RiBookMarkedLine, RiRouteFill } from 'react-icons/ri';
+import { RiBookMarkedLine, RiRouteFill, RiSearchLine } from 'react-icons/ri';
 import { Link } from 'react-router-dom';
 import { LinkButton } from './primitives/button-link';
 
-export const WorkflowListEmpty = () => {
+interface WorkflowListEmptyProps {
+  emptySearchResults?: boolean;
+  onClearFilters?: () => void;
+}
+
+export const WorkflowListEmpty = ({ emptySearchResults, onClearFilters }: WorkflowListEmptyProps) => {
   const { currentEnvironment, switchEnvironment, oppositeEnvironment } = useEnvironment();
 
+  if (emptySearchResults) {
```

Comment:
please follow what we decided about the lists and loading/empty states here: https://novu.slack.com/archives/C07HVLG7G2U/p1737462736756469

Reply:
We have here all the modes as discussed in Sok message, loading, empty search results, empty. Unless I'm missing something we should be good.